README.md

```diff
@@ -1,2 +1,45 @@
 # Hello-Microverse
-This is a trial code review repo
+This is a trial code review repo which is about showing Hello Microverse as a header.
+
+![screenshot](./app_screenshot.png)
```

Comment:
- [ ] The link to the screenshot for your readme file is broken. It will be good to fix this issue because it will make your project even more professional.